From randy.dunlap@oracle.com  Mon May  3 10:51:04 2010
From: Randy Dunlap <randy.dunlap@oracle.com>
Date: Mon, 3 May 2010 09:08:38 -0700
Subject: tty: n_gsm: depends on NET
To: gregkh@suse.de
Message-ID: <20100503090838.e741ef9c.randy.dunlap@oracle.com>

From: Randy Dunlap <randy.dunlap@oracle.com>

n_gsm uses skb functions, so it should depend on NET.

n_gsm.c:(.text+0x123d49): undefined reference to `skb_dequeue'
n_gsm.c:(.text+0x123d98): undefined reference to `kfree_skb'
n_gsm.c:(.text+0x123e1e): undefined reference to `skb_pull'

Signed-off-by: Randy Dunlap <randy.dunlap@oracle.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/char/Kconfig |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/char/Kconfig
+++ b/drivers/char/Kconfig
@@ -284,6 +284,7 @@ config N_HDLC
 config N_GSM
 	tristate "GSM MUX line discipline support (EXPERIMENTAL)"
 	depends on EXPERIMENTAL
+	depends on NET
 	help
 	  This line discipline provides support for the GSM MUX protocol and
 	  presents the mux as a set of 61 individual tty devices.
